src/pages/catalog.js

```diff
@@ -78,50 +66,104 @@ const FeedstockRowAccordionItem = ({ feedstockId, feedstockSpec }) => {
     <>
       {' '}
       {datasets?.length > 0 ? (
-        <AccordionItem>
-          {({ isExpanded }) => (
-            <>
-              <Skeleton isLoaded={!metaIsLoading && !fsIsLoading}>
-                <AccordionButton>
-                  <Box flex='1' textAlign='left'>
-                    {meta?.title}
-                  </Box>
-
-                  {isExpanded ? (
-                    <MinusIcon fontSize='xl' />
-                  ) : (
-                    <AddIcon fontSize='xl' />
-                  )}
-                </AccordionButton>
-
-                <AccordionPanel>
-                  <Text opacity={0.8}>{meta?.description}</Text>
-
-                  <Skeleton isLoaded={!datasetsAreLoading}>
-                    <List spacing={3} my={4}>
-                      {datasets?.map((dataset, index) => (
-                        <DatasetListItem
-                          key={index}
-                          dataset={dataset?.dataset_public_url}
-                        ></DatasetListItem>
-                      ))}
-                    </List>
-                  </Skeleton>
-
-                  <Button
-                    as={Link}
-                    my={4}
-                    href={`/dashboard/feedstock/${feedstockId}`}
-                    colorScheme='teal'
-                    variant='outline'
-                  >
-                    More Details
-                  </Button>
-                </AccordionPanel>
-              </Skeleton>
-            </>
-          )}
-        </AccordionItem>
+        <Box>
+          <Flex alignItems={'center'} justifyContent={'space-between'}></Flex>
+          <Box
+            pos='relative'
+            h='200px'
+            bgImage="linear-gradient(rgba(255, 0, 0, 0.127),rgba(255, 0, 0, 0.1)) , url('')"
```

Comment:
@katamartin, i'm curious... do you have any suggestions about how to programmatically generate these? i would like to have different colors for different datasets, and not have to hard code this here...

Reply:
@andersy005 that sounds great! Are you wanting to use different colors from the theme for the different datasets? If so, it looks like Chakra supports that out-of-the box: https://chakra-ui.com/docs/styled-system/gradient.

Here, this could look something like:
```jsx
<Box
  h='200px'
  bgGradient={`linear(${color}.300, ${color}.100)`}
/>
```
where `color` is a string (e.g. `'red'`) determined from the index of the dataset or pulled randomly.

Reply:
Terrific! thank you for looking into this. I hadn't seen that page... Let me give this a try